docs/src/content/docs/commands/DECR.md

```diff
@@ -101,3 +101,6 @@ OK
 ## Alternatives
 
 You can also use the `DECRBY` command to decrement the value of a key by a specified amount.
+
+## Notes
+- Please refer to the [DECRBY](/commands/decrby) command documentation for more information.
```

Comment:
We can add the information in the Alternatives section and add the hyperlink as follows:
[`DECRBY`](/commands/decrby)